Babylon/groups/api/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 from .commands import list_commands
 from .groups import list_groups
-from ...utils.decorators import require_platform_key
+from Babylon.utils.decorators import require_platform_key
```

Comment:
Ah and Alexis just told me he prefers leaving the dots. It's personal as taste as well.
There are some exotic cases when Babylon.utils will fail (circular imports or so)

Reply:
Just in group init or in each command file ?

Reply:
In each file, sorry, we are planning to add dev tools to uniformize code styling it will help on those matters.

Reply:
Ah you forgot to change this one